src/json/members/healice.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Alice He",
+  "pennkey": "healice",
+  "major": "Computer Science",
+  "school": "Engineering",
+  "bio": "Passionate about the intersection of art and technology, architecture, and Money Heist. Budding college dorm chef. Pesto and hot sauce lover. :P",
+  "hometown": "Washington, DC",
+  "team": "Penn Clubs",
+  "roles": ["Designer"],
+  "photo": "https://imgur.com/uuYavDm",
```

Comment:
This needs to be a direct link to the photo, see the other files in this directory for what the URL looks lik